### ISSUE TITLE
How to realize gradient truncation function in HB Pkg

### ISSUE THREAD

**Issue by taoyun951753** [DeepRec-AI/HybridBackend#94]
# Current behavior
Recently, when I trained the model, the loss function got a Nan value. I want to perform gradient truncation. However HB can only use tf.train.XxxOptimizer api, which is used with  computes_ gradients,  tf.clip_ by_ value(), tf.clip_ by_ Norm() . But now HB uses tf.keras compile, fit mode. The  community tf.keras.optimizers Adam () supports the clipvalue/clipnorm parameter . 

# Expected behavior
I want the HB pkg can support the  gradient truncation function.
# System information
- GPU model and memory:
- OS Platform:
- Docker version:
- GCC/CUDA/cuDNN version:
- Python/conda version:
- TensorFlow/PyTorch version:

# Code to reproduce

```python
```

# Willing to contribute

Yes
